packages/auth/src/providers/cognito/utils/oauth/cancelOAuthFlow.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { OAuthStore } from '../types';
+
+import { createOAuthError } from './createOAuthError';
+
+import { handleFailure } from './index';
+
+export const listenForOAuthFlowCancellation = (store: OAuthStore) => {
+	async function handleCancelOAuthFlow(event: PageTransitionEvent) {
+		const isBackEventWithBfcache = event.persisted;
```

Comment:
Nit: Is this technically misleading naming? I the `persisted` property could in theory indicate forward event, too, right? I'm only nitpicking at this because, in the case we ever need to debug this, if the naming implies something too specific here it could obfuscate some behavior.
```suggestion
		const isBackEventWithBfcache = event.persisted;
```

Reply:
seems the suggestion is the same. Changed to `isBfcache`